contracts/UMA/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -873,21 +907,25 @@ contract PricelessPositionManager is Testable, Lockable {
         view
         returns (address)
     {
-        return
```

Comment:
we can remove this `_getFinancialContractsAdminAddress`  internal function, since it's not being used anymore.